packages/components/src/focal-point-picker/index.js

```diff
@@ -27,7 +27,7 @@ export class FocalPointPicker extends Component {
 		this.state = {
 			isDragging: false,
 			bounds: {},
-			percentages: {},
+			percentages: FocalPointPicker.defaultProps.value,
```

Comment:
To the specific fix however: It's terribly confusing to me that we assign this value and immediately override it in `componentDidMount` with a `setState`.

I think this should just be changed to `props.value` (`props` being the first argument of the `constructor`) and eliminate `componentDidMount` altogether. Otherwise, we're completely disregarding `value` from the `props` for the initial (albeit ultimately wasteful) render.

>You may call setState() immediately in componentDidMount(). It will trigger an extra rendering, but it will happen before the browser updates the screen. This guarantees that even though the render() will be called twice in this case, the user won’t see the intermediate state. Use this pattern with caution because it often causes performance issues. **In most cases, you should be able to assign the initial state in the constructor() instead**. It can, however, be necessary for cases like modals and tooltips when you need to measure a DOM node before rendering something that depends on its size or position.

https://reactjs.org/docs/react-component.html#componentdidmount

_(Emphasis mine)_

Reply:
Done in 9b1b02993b8216bb79b912d5aea3f1cd4e25d12d

Reply:
This looks good to me and tests just fine. Thinking back, I believe I set the percentages in `componentDidMount` because of an earlier approach to determining the image's dimensions, which I moved away from before submitting the initial Pull Request. Agree with @westonruter it's not necessary.